lesson14/js/main.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+//Modal
+const menuLists = document.createElement('ul');
+
+function createModalWindow() {
+	const modalWindow = document.createElement('div');
+	const createMenuContainer = document.createElement('div');
+
+	//modal container
+	modalWindow.classList.add('bl_modal');
+	modalWindow.setAttribute('id', 'js-modalContainer');
+
+	//menu button
+	const createMenuBtn = document.createElement('button');
+	createMenuBtn.setAttribute('id', 'js-loadmenuBtn');
+	createMenuBtn.setAttribute('class', 'btn');
+	createMenuBtn.textContent = `MENU`;
+
+	//close button
+	const createCloseBtn = document.createElement('button');
+	createCloseBtn.setAttribute('id', 'js-closeModal');
+	createCloseBtn.setAttribute('class', 'el_closeModalBtn');
+	createCloseBtn.textContent = '×';
+
+	//input area
+
+	const createInputButton = document.createElement('input');
+	createInputButton.type = 'number';
+	createInputButton.placeholder = 'please input number';
+	createInputButton.min = '1';
+	createInputButton.setAttribute('id', 'number');
+	createInputButton.setAttribute('class', 'el_input');
+
+	createMenuContainer.setAttribute('id', 'js-menu_container');
+	const modal = document.createDocumentFragment();
+
+	modal.appendChild(createInputButton);
+	modal.appendChild(createMenuBtn);
+	modal.appendChild(createCloseBtn);
+	modal.appendChild(createMenuContainer);
+	modalWindow.appendChild(modal);
+	document.body.appendChild(modalWindow);
+
+	loadMenu();
+	closeModalWindow();
+}
+
+//open Modal
+const openModalBtn = document.getElementById('js-openModalBtn');
+
+openModalBtn.addEventListener(
+	'click',
+	() => {
+		const modalBg = document.createElement('div');
+		modalBg.id = 'js-bgFixed';
+		modalBg.classList.add('el_modalBg');
+		document.body.appendChild(modalBg);
+		createModalWindow();
+		document.body.style.position = 'fixed';
+	},
+	false
+);
+
+function loadMenu() {
+	const loadMenuBtn = document.getElementById('js-loadmenuBtn');
+	loadMenuBtn.addEventListener(
+		'click',
+		(event) => {
+			const inputNumber = document.getElementById('number');
+			if (!inputNumber.value === false) {
+				getInputValue();
+				callMenuContents();
+			} else {
+				alert('未入力です。');
+			}
+		},
+		false
+	);
+}
```

Comment:
nits
Is this the width of the indentation?

Reply:
Thank you for your review.
I'm sorry that our response was delayed.

The VScode setting was the default, so I reset it.
Mac is now the main machine!